package/yast2-users.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed May 13 14:04:35 CEST 2020 - schubi@suse.de
+
+- AY proposal: Fixed crash. Showing user settings only because the
+  UI does not fit to the AY settings (bsc#1170815)
```

Comment:
I do not get much this changelog from user POV. What is changed for users? Showing in autoyast user proposal to propose only users and groups?